### PR TITLE
colleciton of helpful hints gleaned from the issue queue.

### PR DESCRIPTION
**Pull request by jmcclelland**
Thanks for the great extension. I spent hours tearing my hair out getting it to work and finally realized it was due to firejail. So, I collected all the things I learned from combing through the issue queue and put them together in the README file in case anyone else has trouble getting this extension to work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,19 @@ stay in the foreground:
 ["gvim", "-f"]
 ```
 
+## Problem solving
```
Comment (jlebon) [inline README.md]: Maybe let's call it "Troubleshooting" ?

File: README.md [hunk 2]
```diff
@@ -136,6 +136,19 @@ stay in the foreground:
...
+
+Some things to try if it doesn't work properly:
+
+ * Ensure you are running the latest version of Firefox or Chrome
```
Comment (jlebon) [inline README.md]: Hmm interesting, does it actually work in Chrome?

File: README.md [hunk 3]
```diff
@@ -136,6 +136,19 @@ stay in the foreground:
...
+ * Try configuring textern to launch using a different short cut
```
Comment (jlebon) [inline README.md]: Minor: for consistency, let's capitalize Textern

**Comment by jmcclelland** [comment]
Thanks for the feedback. I think I incoporated it all. I also expanded the firejail section - my previous suggestion, it turns out, was not enough to get things working in firejail. I'm not sure you want all of that firejail stuff in your README - alternatively, it could just be a note that says: Textern won't work out of the box if you are using firejail - see issue #52 for more info.

**Comment by jlebon** [comment]
Just squashed all the commits into a single one.
Thanks again!

**Pull request merged**